src/bioregistry/data/bioregistry.json

```
@@ -17815,6 +17815,23 @@
     ],
     "uri_format": "https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=$1"
   },
+  "clingen.curation": {
+    "banana": "CCID",
+    "banana_peel": ":",
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "Curations representing evidence aggregation and expert panel assertions based on standardized evaluation procedures",
+    "example": "004126",
+    "github_request_issue": 1414,
+    "homepage": "https://search.clinicalgenome.org/kb/gene-validity",
+    "name": "ClinGen Curation",
+    "pattern": "^\\d{6}$",
+    "uri_format": "https://search.clinicalgenome.org/$1"
```

I think the banana will still have to be added in the URI format here for resolution to work

```suggestion
    "uri_format": "https://search.clinicalgenome.org/CCID:$1"
```